dev-lisp/sbcl/sbcl-2.5.1.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A high performance Common Lisp compiler"
+DESCRIPTION="Steel Bank Common Lisp (SBCL) is a high performance Common \
+Lisp compiler. It is open source / free software, with a permissive license. \
+In addition to the compiler and runtime system for ANSI Common Lisp, it \
+provides an interactive environment including a debugger, a statistical \
+profiler, a code coverage tool, and many other extensions."
+HOMEPAGE="https://sbcl.org"
+COPYRIGHT="2002 Gerd Moellmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+portVersion="2.5.1"
```

Comment:
`portVersion` is already predefined by haikuporter from the version part of the file name, so this isn't needed.

```suggestion
```